src/products/model.ts

```diff
@@ -0,0 +1,17 @@
+import {productsDBConnection} from './db'
+import {DataTypes} from "sequelize";
+import {ProductInstance} from "./types";
+
+const ProductModel = productsDBConnection.define<ProductInstance>("products", {
+    id: {
+        type: DataTypes.INTEGER,
```

Comment:
```suggestion
        type: DataTypes.UUIDV4,
        defaultValue: DataTypes.UUIDV4,
        allowNull: true
```

remove the autoIncrement.
this way the db will automatically generate uuidv4 id's.

it's a standard way for Id's that I've seen.
I'ts better than using a running index number. there are articles we should read to reason about this.